pos_analytic_by_config/models/account_move_line.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        """We need to receive this key from the vals as the create call is encampsulated
+        inside the _get_sale_vals method of the pos.session model."""
```

Comment:
This is not explaining how and where `pos_config_id` context is used. docstring also puts the trailing `"""` in a new line according PEP257.